### PR TITLE
Add AMPHTML Email Hello World

### PR DESCRIPTION
**Pull request by juliantoledo**

### CODE REVIEW

File: src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  <!--
+    Just like in AMP HTML, in AMPHTML Email, most HTML tags can be used directly.
+  -->
+  <h1>Hello World!</h1>
+  <!--
+    Certain tags, such as the `<img>` tag, are replaced
+    with equivalent or slightly enhanced custom AMP HTML tags (see
+    [HTML Tags in the specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md)).
+  -->
+  <amp-img src="/img/amp.jpg" width="400" height="300"></amp-img>
```
Comment (raywainman) [inline src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html]: You'll need to put an absolute path here, the spec doesn't allow relative URLs.
https://ampbyexample.com/img/amp.jpg

File: src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!--
+    The AMPHTML Email boilerplate.
+  -->
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <title>AMPHTML Email Hello World</title>
+  <!-- Import AMP components in the header, for example the `amp-carousel` component. -->
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <!--
+    CSS must be included in a `<style amp-custom>` tag within the header.
```
Comment (sebastianbenz) [inline src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html]: Can you link to the CSS restrictions?

File: src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<!---{
+ "preview": "amp4email",
+ "skipValidation": "true"
+}--->
+<!--
+## Introduction
+
+The AMPHTML Email format provides a subset of AMP components that you can use in email messages. Recipients of AMP emails can view and interact with the AMP components directly in the email message.
```
Comment (sebastianbenz) [inline src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html]: Is AMPHTML Email is official term? Usually, `AMP HTML` is split into two words...

File: src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    Just like in AMP HTML, in AMPHTML Email, most HTML tags can be used directly.
+  -->
+  <h1>Hello World!</h1>
+  <!--
+    Certain tags, such as the `<img>` tag, are replaced
+    with equivalent or slightly enhanced custom AMP HTML tags (see
+    [HTML Tags in the specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md)).
+
+    <p class="mb2">
+      <div class="p1 ampstart-card abe-info"><strong>Note:</strong> URLs most use absolut paths in AMPHTML email.</div>
```
Comment (sebastianbenz) [inline src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html]: s/Note/Important/

File: src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+    Since we imported the `amp-carousel` component in the header as example, let's try it here.
+  -->
+  <amp-carousel width="400" height="300" type="slides">
+    <amp-img src="https://ampbyexample.com/img/image1.jpg" width="400" height="300" alt="a sample image"></amp-img>
+    <amp-img src="https://ampbyexample.com/img/image2.jpg" width="400" height="300" alt="another sample image"></amp-img>
+    <amp-img src="https://ampbyexample.com/img/image3.jpg" width="400" height="300" alt="and another sample image"></amp-img>
+  </amp-carousel>
+  <!--
+    ### Allowed CSS properties and selectors
```
Comment (sebastianbenz) [inline src/amphtml-email/10_Introduction/_AMPHTML_Email_Hello_World.html]: Suggestion: move this to the CSS block above.

**Comment by juliantoledo** [comment]
@sebastianbenz Fixed comments, thank you.

File: tasks/compile-example.js [hunk 6]
```diff
@@ -273,7 +273,7 @@ module.exports = function(config, indexPath, updateTimestamp) {
         compileTemplate(stream, example, args, {
           template: previewTemplate,
           targetPath: example.targetPreviewPath(),
-          postProcessors: [replaceAmpAdRuntime, replaceAmpStoryRuntime],
+          postProcessors: [replaceAmpAdRuntime, replaceAmpStoryRuntime, replaceAmpHtmlEmailRuntimeAddViewport],
```
Comment (sebastianbenz) [inline tasks/compile-example.js]: Can you do me a favour and extract all post processors into a list, this way it becomes easier to add more. Thanks!

Comment (juliantoledo) [reply tasks/compile-example.js]: done

**Comment by juliantoledo** [comment]
Hi @spacedino,

Sebastian mentioned that it would be a good idea to get your feedback in the new AMPHTML intro, we have it in staging here:
https://amp-by-example-staging.appspot.com/amphtml-email/introduction/amphtml_email_hello_world/

and we also created a new preview mode for emails:
https://amp-by-example-staging.appspot.com/amphtml-email/introduction/amphtml_email_hello_world/preview/

Thank you!

**Pull request merged**